helm/calico-app/templates/rbac.yaml

```diff
@@ -100,6 +100,15 @@ rules:
       - namespaces
     verbs:
       - get
+  {{- if eq .Values.provider "azure" }}
```

Comment:
For my curiosity, why do we need this?

Reply:
`# Needed when configuring bgp password in bgppeer`
Source: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/templates/addons/calico.yaml#L3389

Reply:
It might be helpful to retain that comment

Reply:
Done